test/system/710-kube.bats

```diff
@@ -19,15 +20,18 @@ json.dump(yaml.safe_load(sys.stdin), sys.stdout)'
 ###############################################################################
 # BEGIN tests
 
+# bats test_tags=ci:parallel
```

Comment:
All tests in this file have the `ci:parallel` tag. I am tempted to remove them all, and add it at top using `file_tags`, because (IMO) the tests here _should_ always be parallel-safe, including future additions. IOW I prefer this risk:
* new test gets added, it runs (by default) in parallel but somehow it was not written properly nor reviewed properly and we get new flakes
over:
* new test gets added, but we all forget to add/review the `ci:parallel` tag, so it runs serial by default, slowing down the world.

Opinions welcome.

Reply:
I would default to parallel.  Flakes might show up, but we should be able to find them quickly.
Podman by default runs i parallel by design, so better to default to it.